lib/type/date.js

```diff
@@ -0,0 +1,93 @@
+module.exports = (function ( ƒ ){
+
+    var date = ƒ.Date = ƒ.date = {},
+    replaceChars = {
```

Comment:
Since `replaceChars` is only used on `.format` method, could you please save it under `date.format.chars` ?